pkg/internal/apis/certmanager/validation/issuer_test.go

```diff
@@ -640,30 +640,54 @@ func TestValidateACMEIssuerDNS01Config(t *testing.T) {
 			},
 			errs: []*field.Error{},
 		},
-		"valid rfc2136 config": {
+		"rfc2136 provider with missing nameserver": {
+			cfg: &cmacme.ACMEChallengeSolverDNS01{
+				RFC2136: &cmacme.ACMEIssuerDNS01ProviderRFC2136{},
+			},
+			errs: []*field.Error{
+				field.Required(fldPath.Child("rfc2136", "nameserver"), ""),
+			},
+		},
+		"rfc2136 provider with IPv4 nameserver": {
 			cfg: &cmacme.ACMEChallengeSolverDNS01{
 				RFC2136: &cmacme.ACMEIssuerDNS01ProviderRFC2136{
 					Nameserver: "127.0.0.1",
 				},
 			},
 			errs: []*field.Error{},
 		},
-		"missing rfc2136 required field": {
+		"rfc2136 provider with IPv6 nameserver": {
```

Comment:
Can we add a test for an unenclosed IPv6?

Reply:
Done :)